core/src/main/resources/i18n/displayStrings.properties

```
@@ -945,9 +947,9 @@ portfolio.pending.error.depositTxNotConfirmed=The deposit transaction is not con
   For further help please contact the Bisq support channel at the Bisq Keybase team.
 
 portfolio.pending.support.headline.getHelp=Need help?
-portfolio.pending.support.text.getHelp=If you have any problems you can try to contact the trade peer in the trade \
-  chat or ask the Bisq community at https://bisq.community. \
-  If your issue still isn't resolved, you can request more help from a mediator.
+portfolio.pending.support.text.getHelp=Take a look at the trade process \
+  [here](https://docs.bisq.network/getting-started.html#4-send-payment)\
+  \n\nIf you have any problems you can try to contact the trade peer in the trade chat.
```

```suggestion
portfolio.pending.support.text.getHelp=Many issues can be resolved by simply chatting with your trading peer.
```

It would be great if there was an empty line under the "Need help?" heading.